### DISCUSSION TITLE
Current firmware (NOT FOR ROUTINE UPDATES)

### DISCUSSION THREAD

**Discussion by hirnlego** [Befaco/Oneiroi#47] [Announcements]
This is the current firmware of the module:

[Oneiroi.zip](https://github.com/user-attachments/files/18062110/Oneiroi.zip)

**Comment by Diegoterrores** [comment]
DO NOT USE THIS FIRMWARE FOR ROUTINE UPDATES! 
TO UPDATE TO PATCH 1.1, [USE THIS LINK:](https://github.com/Befaco/Oneiroi/releases/tag/v1.1
)



Just to clarify:
the Oneiroi software architecture consists of three distinct layers: Bootloader, Firmware, and Patch. The Firmware and Bootloader layers ensure that the module is compatible with the Owl platform. They manage essential functions, such as reading buttons and knobs, while the Patch layer contains the DSP code that defines the Oneiroi experience.
This is just the firmware, which usually does not need to be updated by the end user.
